### DISCUSSION TITLE
Add Cinnamon Menu Editor to Menu by default

### DISCUSSION THREAD

**Discussion by TranzistorP** [linuxmint/discussions#242] [Ideas]
Currently, by default, the Menu Editor only opens
'Right click on menu launch button - Edit menu'
![image](https://github.com/linuxmint/discussions/assets/96445511/5dac522a-bf13-4f01-bf97-a7843326d10b)
To figure out how to edit the Menu, I first spent 20 minutes searching for this function in Menu and System Settings.
Then i googled it but a Google search in my language only suggested using third-party programs, and only an search in English returned result.

Although in the Menu Editor you can then easily include it in the Menu.
![image](https://github.com/linuxmint/discussions/assets/96445511/d4a304b3-532b-4d99-a66f-cf1c6e6686b7)
I think it would be handy to add the Menu Editor to Menu and perhaps System Settings by default.
![image](https://github.com/linuxmint/discussions/assets/96445511/09e13f12-b7a9-4914-b6f5-4c0392a627ad)
Sorry for my English.

**Comment by TranzistorP** [comment]
I have researched this issue. This parameter is set in the file /usr/share/applications/cinnamon-menu-editor.desktop,
```
[Desktop Entry]
Name=Cinnamon Menu Editor
Comment=Edit the entries of the Cinnamon menu applet
Exec=cinnamon-menu-editor
Icon=alacarte
OnlyShowIn=X-Cinnamon;
Terminal=false
Type=Application
NoDisplay=true
Categories=GNOME;GTK;Settings;DesktopSettings;
StartupNotify=false
```

,the line NoDisplay=true is responsible for this behavior. I wonder if this is a design or were there any problems?

**Comment by kacperpaczos** [comment]
Maybe, It should be in system settings? Create issuse in cinnamon center.
